os/include/tinyara/audio/alc5658.h

```diff
@@ -120,10 +116,6 @@
 #error CONFIG_ALC5658_INFLIGHT must fit in a uint8_t
 #endif
 
-#ifndef CONFIG_ALC5658_MSG_PRIO
-#define CONFIG_ALC5658_MSG_PRIO          1
-#endif
-
 #ifndef CONFIG_ALC5658_BUFFER_SIZE
 #define CONFIG_ALC5658_BUFFER_SIZE       8192
```

Comment:
this value should change 8192 to 2048.
is that correct?

Reply:
No, if no value is assigned, then better to keep the default value as 8k.